.rubocop.yml

```diff
@@ -235,6 +235,10 @@ Naming/BlockForwarding:
 Style/ArgumentsForwarding:
   RedundantBlockArgumentNames: []
 
+Lint/MissingSuper:
+  Exclude:
+    - 'lib/wca_exceptions.rb'
+
```

Comment:
Veeeeeery unhappy with this. Strong bias towards finding a better solution that doesn't need a Rubocop rule, unless it's unreasonably painful and costs you days upon days.

Reply:
agree! I'm pretty sure this is easily done

Reply:
Ok I've fixed it - I admit I was being lazy